addons/debugger/userscript.js

```diff
@@ -162,7 +162,8 @@ export default async function ({ addon, console, msg }) {
   window.addEventListener("resize", () => {
     moveInterface(lastX, lastY);
   });
-  interfaceHeader.addEventListener("mousedown", handleStartDrag);
+  interfaceHeader.addEventListener("pointerdown", handleStartDrag);
+  interfaceHeader.addEventListener("touchstart", (e) => e.preventDefault());
```

Comment:
e.preventDefault() is too broad here. Will prevent user being able to click the header buttons.
```suggestion
  interfaceHeader.addEventListener("touchstart", (e) => {
    // Only prevent default if not touching a button/interactive element
    if (!e.target.closest('button, [role="button"], .sa-debugger-tabs *, .sa-debugger-header-buttons *')) {
      e.preventDefault();
    }
  });
```

Reply:
Whoops!

Changing this to a `"touchmove"` listener also fixes this.